config.yaml

```diff
@@ -137,7 +137,7 @@ options:
       The database name can be changed but must be created.
     type: string
   patch-storage.filesystem-path:
-    default: "/home/livepatchu/.livepatch/patches"
+    default: "/var/lib/livepatch/patches"
```

Comment:
shouldn't we address this change somewhere in the doc?

Reply:
We don't currently support setting up a k8s volume for storing patches on the filesystem so no doc for this exists yet.

Reply:
Yeah, if we point to this path in any doc, we should update that. I don't remember where, though.

Reply:
it is not a problem for k8 but could it be a problem for snap deployment?